internal/hcsoci/create.go

```diff
@@ -229,3 +229,19 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 	}
 	return system, resources, nil
 }
+
+func (coi *createOptionsInternal) isV2Xenon() bool {
```

Comment:
We really just need to remove all the v1 schema support, but this is more readable at least :)

Reply:
It would be good to add comments here explaining what concepts we are checking. I'd like to use "V1" and "V2" more carefully.